portality/view/account.py

```diff
@@ -250,6 +259,7 @@ def register():
         account = models.Account.make_account(
             username=form.w.data,
             email=form.n.data,
+            roles=[r.strip() for r in form.roles.data.split(',')]
         )
         account.set_password(form.s.data)
         account.save()
```

Comment:
Novel to me :). I think WTForms has had support for a long time but I'm used to just using 2 args with it (which can be a pain) so that was probably more so me learning about this!